lms/djangoapps/course_blocks/transformers/helpers.py

```diff
@@ -0,0 +1,38 @@
+"""
+Transformers helpers functions.
+"""
+from openedx.core.djangoapps.user_api.partition_schemes import RandomUserPartitionScheme
+from openedx.core.djangoapps.course_groups.partition_scheme import CohortPartitionScheme
+# TODO 8874: Make it so we support all schemes instead of manually declaring them here.
+INCLUDE_SCHEMES = [CohortPartitionScheme, RandomUserPartitionScheme, ]
```

Comment:
This can be removed, as long as all schemes have get_group_for_user() implemented (currently the case).